.travis.yml

```diff
@@ -10,19 +10,22 @@ before_install:
   - curl -L -s https://github.com/golang/dep/releases/download/v${DEP_VERSION}/dep-linux-amd64 -o $GOPATH/bin/dep
   - chmod +x $GOPATH/bin/dep
   - travis_retry dep ensure
+  - curl -LO https://git.io/get_helm.sh && chmod 700 get_helm.sh && ./get_helm.sh
 
 deploy:
   - provider: script
     skip_cleanup: true
     script: make travis-latest-deploy
     on:
-      repo: redhat-cop/cert-utils-operator
-      branch: master
+      #repo: redhat-cop/cert-utils-operator
```

Comment:
This will cause issues on branches where integration with a container registry is not enabled

Reply:
@sabre1041 you are right. I needed those settings to test on my fork/branch. anyway I restored the previous settings. push is on the way.